packages/dx-react-grid/docs/guides/editing.md

```diff
@@ -42,6 +43,20 @@ Note, you can also use the `onAddedRowsChange` event to initialize a created row
 
 .embedded-demo({ "path": "grid-editing/edit-row-controlled", "showThemeSelector": true })
 
+## Inline Cell Editing
+
+### Basic
+
+You can use `TableInlineCellEditing` plugin to edit single cell. With this plugin you turn cell in editing state by click. When you press `Enter` or lost focus from cell (for example click on another cell or element) changes aplly. When you press 'Escape' changes cancel.
```

Comment:
You use ` with Enter and ' with Escape. I think we should use one type of quotes. I prefer the first